lib/googleanalytics.jsx

```diff
@@ -0,0 +1,27 @@
+var React = require('react');
+var config = require('./config');
+
+var GoogleAnalyticsBasicTag = React.createClass({
+  render: function() {
+    if (config.GA_ACCOUNT) {
+      return (
+        <script dangerouslySetInnerHTML={{
+          __html: "(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){ \
```

Comment:
Oh, sorry, I missed this somehow... Nope, it won't reload `analytics.js` on every page load if we do it at the beginning of `run()`... The `run()` function only gets called once per initial page load, while the callback passed to `Router.run()` is called once on initial page load, and then again every time the user navigates to a new page (if they're using hash-based or pushState navigation). I actually changed the code to work this way in bd0a8b3b15276d31615e57f0f4501a790790e168, apologies for not asking for code review from you first. If you think there's anything amiss there I will revert the commit immediately.